src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -151,6 +151,8 @@ static bool updateForward
   std::string     mimeType     = "application/json";
   std::string     out;
   int             r;
+  int             providerLimit  = DEFAULT_PAGINATION_LIMIT_INT;
+  int             providerOffset = DEFAULT_PAGINATION_OFFSET_INT;
```

Comment:
Similar comment to the one done at src/lib/ngsiNotify/doNotify.cpp: 

Remove this block and use -1 for providerLimit and provideOffset in the call to httpRequestSend() below, meaning with <0 value that the parameter hasn't to be included.

Reply:
Fixed in 1e47b07